Docs: Render the filters in a collapsible panel

VasasA opened the pull request:
To render the filters above the table content in a collapsible panel, you may use: `layout: Layout::AboveContentCollapsible`

VasasA committed:
Docs: Render the filters in a collapsible panel

Render the filters above the table content in a collapsible panel.

zepfietje commented:
Thanks for the PR, @VasasA!

Could you update the example to be similar to the ones above it?

VasasA committed:
Docs: Example is updated for filters in a collapsible panel.

Docs: Example is updated for filters in a collapsible panel.
PR is ready for review.

VasasA commented:
😃  You are welcome, @zepfietje 
Example is updated.

zepfietje commented:
Perfect, thanks!
